kv/redis/redis.go

```diff
@@ -230,52 +226,59 @@ func (r *Redis) DeleteTree(directory string) error {
 // Pass previous = nil to create a new key.
 // NOTE: we haven't introduce script on this page yet.
 // We perform read-modify-write on two separated operations which is not atomic guaranteed
-func (r *Redis) AtomicPut(key string, value []byte, previous *store.KVPair, options *store.WriteOptions) (bool, *store.KVPair, error) {
+func (r *Redis) AtomicPut(key string, value []byte, previous *store.KeyValue, options ...store.PutOption) (*store.KeyValue, error) {
 	existedOne, err := r.Get(key)
 	if err != nil && err != store.ErrKeyNotFound {
-		return false, nil, err
+		return nil, err
 	}
 	if previous == nil && existedOne != nil {
-		return false, nil, store.ErrKeyModified
+		fmt.Println("exist object: ", existedOne)
```

Comment:
We have a `Logger` interface, I suggest using that.
Under `github.com/getamis/sirius/log`.

Reply:
this is redundant code for testing, i will remove